### ISSUE TITLE
Intermediate HTML & CSS: Translate Animation from English to Russian

### ISSUE THREAD

**Issue by PhilipHarney** [CoderDojo-Content/content-hack#115]
**If this is your first time working on *Intermediate HTML & CSS* then you will need to be added as a collaborator. You can find instructions on how to do this at http://dojo.soy/gitbook-collab**
1. Make sure you have assigned this task card to yourself and moved it to the **In Progress** column.
2. Go to https://www.gitbook.com/book/coderdojo/intermediate-html-css-sushi/edit and create a change request named **Translation of Animation (animations.md) to Russian**. See how to do this at http://dojo.soy/gitbook-makecr
3. Switch the view to русский ru. See how to do this at http://dojo.soy/gitbook-changelang
4. Translate the English text into Russian. For an overview of what to translate, see http://dojo.soy/translation-guidelines.
5. Once the translation is complete, save your work by clicking the **Publish** button.
6. Move this task card into the **Review** column.
7. If you have time, pick another task from the **Next** column!
